test/libyul/ewasmTranslationTests/invalid.yul

```diff
@@ -0,0 +1,13 @@
+{
+  sstore(0, 1234)
+  mstore(1, 0)
+  invalid()
+  sstore(0, 5678)
+  mstore(1, 0xff)
+}
+// ----
+// Trace:
+// Memory dump:
+//     20: 00000000000000000000000000000000000000000000000000000000000004d2
+// Storage dump:
+//   0000000000000000000000000000000000000000000000000000000000000000: 00000000000000000000000000000000000000000000000000000000000004d2
```

Comment:
Does the interpreter correctly evaluate `invalid()`?

Reply:
I've realised it too that with decimals it is hard to follow. Changed to hex and it confirms to be working.